src/main/java/org/wso2/carbon/connector/ldap/SearchEntry.java

```diff
@@ -139,13 +171,34 @@ private SearchResult makeSureOnlyOneMatch(NamingEnumeration<SearchResult> result
         }
     }
 
+    protected byte[] swapBytes(byte[] bytes) {
```

Comment:
Please add a method level comment on what we are doing at this point?

Reply:
Done